remotemonitor/monitor.go

```diff
@@ -76,7 +76,12 @@ func NewMonitor(cfg Config) (*Monitor, error) {
 		m.ServeHTTP(w, req)
 	}), m.tw)
 	mux := http.NewServeMux()
-	mux.HandleFunc("/health", func(w http.ResponseWriter, req *http.Request) { io.WriteString(w, "ok") })
+	mux.HandleFunc("/health", func(w http.ResponseWriter, req *http.Request) {
+		if _, err := io.WriteString(w, "ok"); err != nil {
+			w.WriteHeader(http.StatusInternalServerError)
+			return
+		}
```

Comment:
Same as elsewhere

Reply:
sure thing.  I simply copied the err pattern from earlier in the same function.  I've now updated it to ignore the error.